iib/workers/tasks/build.py

```diff
@@ -1431,3 +1469,43 @@ def _add_ocp_label_to_index(ocp_version, temp_dir, dockerfile_name):
         log.debug(
             'Added the following line to %s: %s', dockerfile_name, label,
         )
+
+
+def _add_distribution_scope_label_to_index(distribution_scope, temp_dir, dockerfile_name):
```

Comment:
This is basically a copy-paste of `_add_ocp_label_to_index`. Why not make a generic function called `add_label_to_index` that accepts the label name and value?